### ISSUE TITLE
More details about training process

### ISSUE THREAD

**Issue by Li-dacheng** [Rappsilber-Laboratory/AlphaLink2#15]
Can you provide more details about your training, such as the source of the training data? Is it necessary for the crosslink data to represent a 25A distance between two proteins in a complex? Also, could you share the training script?

Thank you for your work; it's truly a significant breakthrough.

**Comment by u-lupo** [comment]
Hi @lhatsk.  Sorry for writing on a very old thread, but just to be sure: did you rather use the fine-tuning Uni-Fold script here https://github.com/dptech-corp/Uni-Fold/blob/main/finetune_multimer.sh?  The `train_multimer` one you shared seems to be for training from scratch.  And, did you use the same parameters as in this script?

**Comment by lhatsk** [comment]
We only fine-tuned the network in the paper due to limited resources. I used the finetune_multimer.sh script you linked. Same parameters just removed the warmup steps.
